pkg/endpoints/endpoints.go

```diff
@@ -144,6 +150,16 @@ func FetchRCEndpoint(serviceEndpoint []ServiceEndpoint) string {
 	return ""
 }
 
+// FetchEndpoints returns the endpoint associated with serviceID otherwise empty string.
+func FetchEndpoints(serviceID string, serviceEndpoint []ServiceEndpoint) string {
```

Comment:
I see FetchVPCEndpoint can't be used in this because that has a completely a different params and logic, lets mention that here in the comment.